infrastructure-playbooks/templates/logfile.j2

```diff
@@ -0,0 +1,11 @@
+LVS output:
+-----------
+{{ lvs_out.stdout }}
+
+Sample output for osds.yml:
+----------------------------
+{% for bucket_index in data_lvs %}
```

Comment:
isn't your variable ``lvs`` and not ``data_lvs``? We should probably not use ``bucket_index`` here either, but that's just a nitpick.